src/app/form-page/form-page.component.ts

```diff
@@ -0,0 +1,273 @@
+import { Component, NgZone, OnInit, ViewChild } from "@angular/core";
+import { Form } from "../models/form";
+import { FormBuilder, Validators, FormControl } from "@angular/forms";
+
+import { Person } from "../models/person";
+import { Answer } from "../models/answer";
+import { AnswerPerson } from "../models/answer-person";
+import { FormService } from "../form-service/form.service";
+import { CdkTextareaAutosize } from "@angular/cdk/text-field";
+import { take } from "rxjs/operators";
+
+@Component({
+  selector: "app-form-page",
+  templateUrl: "./form-page.component.html",
+  styleUrls: ["./form-page.component.css"]
+})
+export class FormPageComponent implements OnInit {
+  constructor(
+    private fb: FormBuilder,
+    private formService: FormService,
+    private ngZone: NgZone
+  ) {}
+
+  @ViewChild("autosizeFirst") autosizeFirst: CdkTextareaAutosize;
```

Comment:
Use meaningful names. For example hobbiesResizableArea or something like that. Just first, second, third etc. don't give much meaning. Imagine if someone switched something places in UI and then you would have to guess which variables is what in UI.

Reply:
This is fixed and commited to frontend-code-fixes.